_clients/javascript/helpers.md

```diff
@@ -11,7 +11,7 @@ Helper methods simplify the use of complicated API tasks. For the client's compl
 
 ## Bulk helper
 
-The bulk helper simplifies making complex bulk API requests.
+The bulk helper simplifies making complex bulk API requests. (Alternatively, you can use `client.bulk` which allows multiple types of operations, for example, you can mix `delete` with `index`. For more information, see https://github.com/opensearch-project/opensearch-js/blob/main/guides/bulk.md.)
```

Comment:
I would create a section named "bulk operations" in the main javascript client page and specify that there are 2 ways to do bulk operations there.

Reply:
I think we want the user to look at `client.helpers.bulk` first because it fits 99% of the use cases and is a lot more streamlined to use than `client.bulk`.  So, it makes sense that `client.bulk` takes a backseat as most users should not be bothered with it.

Reply:
> I would create a section named "bulk operations" in the main javascript client page and specify that there are 2 ways to do bulk operations there.

I will tackle that in a separate PR, since we already have a way to link to the helper page from the opening paragraph in https://opensearch.org/docs/latest/clients/javascript/index/.